### PR TITLE
LoW: some fixes for S10, S22

### PR DESCRIPTION
**Pull request by amakriLexa04**
Well, I was bored and decided to fix a few small bugs in LoW. I can't say that I did my best, but I got the job done.

List of fixed issues:
[LoW S9 and S10: Scenario designs are too similar #7137 ](https://github.com/wesnoth/wesnoth/issues/7137)
[LoW: Clarify player's veterans joining Landar #8359 ](https://github.com/wesnoth/wesnoth/issues/8359) 
[LoW S22: Enemy camp starts too close to friendly camp #8360 ](https://github.com/wesnoth/wesnoth/issues/8360)

### CODE REVIEW

**Comment by amakriLexa04** [comment]
I'm sorry about that. I'll name the next commits properly

**Comment by soliton-** [comment]
No worries. There is just a single commit so the commit message can also be adjusted when doing a squash merge.

**Review by Dalas121 — CHANGES_REQUESTED** [review]
Thanks for working on this! Couple grammar things:

Can you change: _"Some of Kalenz’s veterans joined the host of the Northern Elves under Uradredia, but most have chosen to be traitors and join Landar. Very soon, enemy army appeared..."_
To: _"Though some of Kalenz's veterans had joined Landar in revolt, others instead joined the northern host under Uradredia. Very soon, the Landar’s army appeared..."_

And can you change: _"wanting to stow their weapons and who now stands with Uradredia"_
To: _"to stand with Uradredia"_

**Comment by amakriLexa04** [comment]
Done

Also, I will be happy to help with anything else related to WML

**Comment by Dalas121** [comment]
Thanks!

Ack, I made a typo.  _"the Landar’s"_ should just be _"Landar’s"_.  Sorry about that!

**Review by Dalas121 — APPROVED** [review]

**Comment by amakriLexa04** [comment]
Fixed this typo

**Comment by stevecotton** [comment]
@Dalas121 is there anything else to do here?

Reminder to squash-merge the PR.

File: data/campaigns/Legend_of_Wesmere/scenarios/chapter5/22_Northern_Battle.cfg [hunk 1]
```diff
@@ -30,7 +30,7 @@
             story= _ "It was clear to Kalenz and Cleodil that there would be no sanctuary for them while Landar lived. But during their years in seclusion the fame of their deeds had only grown. Many healers and sorceresses responded to Cleodil’s call, and Kalenz found that recruits came to him as readily as if he were still High Lord in name."
         [/part]
         [part]
-            story= _ "Kalenz’s veterans joined the host of the Northern Elves under Uradredia. Very soon, Landar’s army appeared..."
+            story= _ "Though some of Kalenz's veterans had joined Landar in revolt, others instead joined the northern host under Uradredia. Very soon, Landar’s army appeared..."
```
Comment (soliton-) [inline data/campaigns/Legend_of_Wesmere/scenarios/chapter5/22_Northern_Battle.cfg:33]: This replaces the proper right quotation mark with an apostrophe.

**Comment by Dalas121** [comment]
Squashed. Let me know if the commit message was OK, or if I should have changed it.

**Comment by soliton-** [comment]
The commit message github proposes is never good. No point in writing 3 times the same thing for example.

Would have been nice if the comment I made would have been acted on before merging.

**Comment by Dalas121** [comment]
Oh, I misunderstood your comment. Sorry about that; I'll fix that apostrophe.

**Comment by Dalas121** [comment]
Ah, nevermind, looks like you fixed it already.

**Pull request merged**